www/content/scoop.md

```diff
@@ -31,6 +31,9 @@ scoop:
     name: goreleaserbot
     email: goreleaser@carlosbecker.com
 
+  # The project name and current git tag are used in the format string.
+  commit_template: "Scoop update for {{ .ProjectName }} version {{ .Tag }}"
```

Comment:
maybe rename it to `commit_msg_template`? 🤔

Reply:
Renamed as requested.